### DISCUSSION TITLE
Multi-Task learning

### DISCUSSION THREAD

**Discussion by austinmw** [open-mmlab/mmdetection#10123] [Q&A]
Hi, I'm interested in attempting to set up multi-task learning with object detection + super-resolution. I'm curious if anyone else has done or thought about doing multi-task learning with mmdet, and has any tips for accomplishing it? 

I see that mmpretrain has a MultiTaskDataset and MultiTaskHead, so am planning to investigate those more to understand their setup.

**Comment by Felix-fz** [comment]
Hi, I do the same work. Did you now use which one?

**Reply by XJWang628** [reply]
Hi, have you successfully implemented this setup using mmdetection/mmpretrain components?
